mockwebserver-junit5/src/test/java/mockwebserver3/junit5/ExampleMockWebServerParameterTest.kt

```diff
@@ -0,0 +1,15 @@
+package mockwebserver3.junit5
+
+import mockwebserver3.MockWebServer
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.RegisterExtension
+import kotlin.test.assertTrue
+
+@ExtendWith(MockWebServerExtension::class)
```

Comment:
Can we register our extension using ServiceLoader instead? That way the annotation is unnecessary?

Reply:
Yes, I've added an example of that now.